lib/policies.py

```diff
@@ -513,7 +513,6 @@ def updateCurrentHostList(self, currentPrivateIps, currentPublicIps, host, publi
             self.populateHosts(currentPrivateIps, host, node_type)
             if len(public_host.strip()) != 0:
                 self.populateHosts(currentPublicIps, public_host, node_type)
-                currentPublicIps.append(public_host)
```

Comment:
Should we not have an alternative to `append` for the dictionary type here? Is `public_ip` getting added to the `currentPublicIPs` someplace else?
Also, please do grep on the source to see if we are missing this in any other places.

Reply:
The alternative to append is `populateHosts()` which is called in line 515 above.